test/Clownface/out.js

```diff
@@ -65,4 +68,183 @@ describe('.out', () => {
 
     assert.strictEqual(result._context.length, 2)
   })
+
+  describe('with language option', () => {
+    const testData = turtle`${ex.ananas}
+      ${rdfs.label} "Pineapple" ;
+      ${rdfs.label} "Ananas"@pl ;
+      ${rdfs.label} "Ananas"@de ;
+      ${rdfs.label} "Ananász"@hu ;
+      ${rdfs.label} "Ananas"@sr-Latn ;
+      ${rdfs.label} "Ананас"@sr-Cyrl ;
+    .
+    
+    ${ex.noLabels} ${rdfs.label} _:foo , ${ex.bar}, 41 .
+    
+    ${ex.apple}
+      ${rdfs.label} "Apple"@en ;
+      ${rdfs.label} "Apfel"@de ;
+      ${rdfs.label} "Јабука"@sr-Cyrl .
+      
+    ${ex.carrot}
+      ${rdfs.label} "Karotte"@de ;
+      ${rdfs.label} "Karotte"@de-AT ;
+      ${rdfs.label} "Rüebli"@de-CH ;
+    .
+    
+    ${ex.eggplant}
+      ${rdfs.label} "Psianka podłużna"@pl, "Bakłażan"@pl, "Oberżyna"@pl .
+      
+    ${ex.kongressstrasse} 
+      ${rdfs.label} "Kongressstraße"@de ;
+      ${rdfs.label} "Kongreßstraße"@de-DE-1901 ;
+    .`.toString()
+
+    describe('filtered by single language parameter', () => {
+      it('should not return non-literals and non-string literals when language parameter is defined', async () => {
+        // given
```

Comment:
I totally agree on having the pattern for the tests, but does it need the comments? I try to have only blank lines between the blocks. I think that works already pretty well to separate the blocks while reading the code.